evalml/tests/component_tests/test_prophet_regressor.py

```diff
@@ -0,0 +1,79 @@
+import pytest
```

Comment:
I think we need coverage for the `X=None` case, which would require tweaking how `build_prophet_df` works.